### ISSUE TITLE
RemoveUnused gives spurious --help error when -Ywarn-unused is enabled

### ISSUE THREAD

**Issue by ches** [scalacenter/scalafix#933]
```
mkdir -p /tmp/scalafix-test/project
cd /tmp/scalafix-test

echo 'addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.1")' > project/plugins.sbt
echo 'sbt.version = 1.2.8' > project/build.properties
echo 'rules = [RemoveUnused]' > .scalafix.conf

cat << EOF > build.sbt
name := "scalafix-test"

scalaVersion := "2.12.8"
scalacOptions += "-Ywarn-unused"
EOF
```

```
$ sbt --warn
sbt:scalafix-test> scalafixEnable
[info] Set current project to scalafix-test (in build file:/private/tmp/scalafix-test/)
sbt:scalafix-test> scalafix --help
error: The Scala compiler option "-Ywarn-unused" is required to use RemoveUnused.
To fix this problem, update your build to use at least one Scala compiler
option that starts with "-Ywarn-unused"
[success] Total time: 2 s, completed Jan 20, 2019 2:59:32 PM
```

None of `-Ywarn-unused:_`, `-Ywarn-unused:imports`, or `Ywarn-unused-import` makes a difference. Though `scalafix --check RemoveUnused` or `scalafix RemoveUnused` work just fine.

**Comment by olafurpg** [comment]
Thanks for reporting! I am able to reproduce. This looks like an oversight in the logic for compiler option validation when running `--help`.

**Comment by ches** [comment]
This was resolved by #983, thanks @tanishiking!
